user.go

```diff
@@ -0,0 +1,23 @@
+package mockoidc
+
+// User defines the expected user values
+type User struct {
```

Comment:
I added a dummy `user.go` file when working on `MockOIDC` methods that needed a `*User` -- feel free to clobber it with yours when you rebase, yours is much more implemented